firstvoices/backend/serializers/async_results_serializers.py

```diff
@@ -5,7 +5,36 @@
 from drf_spectacular.utils import extend_schema_field
 from rest_framework import serializers
 
-from backend.models.async_results import CustomOrderRecalculationResult
+from backend.models.async_results import (
+    BulkVisibilityJob,
+    CustomOrderRecalculationResult,
+)
+from backend.serializers.base_serializers import (
+    BaseSiteContentSerializer,
+    base_timestamp_fields,
+)
+from backend.serializers.fields import SiteHyperlinkedIdentityField
+
+
+class BaseJobSerializer(BaseSiteContentSerializer):
```

Comment:
(Optional) Mybe we can move the BaseJob serializer to a separate file, BaseJobSerializer.py or something since its not related to the other classes, and then can be imported or used here ?

Reply:
I've renamed the `async_results` files to `job` files to make the connection more clear